test-site/README.md

```diff
@@ -0,0 +1,53 @@
+# Test site
+Add a url to a game session in the field and press "Go to site", will show that game/site in the iframe available. Any events sent by the frame will be displayed in the top part. The top three have data sent by the iframe, the ones below that are just the events, without any data. If they are green/have a green check box it is successful. If there is an orange checkbox, the event was sent, but something is wrong with the data. The top 
```

Comment:
```suggestion
Add a url to a game session in the field and press "Go to site", will show that game/site in the iframe available. Any events sent by the frame will be displayed in the top part. The top three have data sent by the iframe, the ones below that are just the events, without any data. If they are green/have a green check box it is successful. If there is an orange checkbox, the event was sent, but something is wrong with the data.
```

Reply:
Noticed some other things in the text, so i'll update those as well